asistente_ladm_col/gui/supplies/dlg_missing_snc_supplies.py

```diff
@@ -0,0 +1,309 @@
+                                                                   ".<br/><br/>If you run the function, the data will be overwritten.<br/><br/>Do you still want to continue?"),
+                QMessageBox.Yes, QMessageBox.No)
+
+            return reply
+        else:
+            reply = QMessageBox.Yes
+
+        if reply == QMessageBox.Yes and os.path.isfile(self.xlsx_path):
+            reply = QMessageBox.question(self,
+                QCoreApplication.translate("MissingSncSuppliesDialog", "Warning"),
+                QCoreApplication.translate("MissingSncSuppliesDialog", "The <b>xlsx</b> already exist in the folder"
+                                                                   ".<br/><br/>If you run the function, the data will be overwritten.<br/><br/>Do you still want to continue?"),
+                QMessageBox.Yes, QMessageBox.No)
+
+            return reply
+        else:
+            reply = QMessageBox.Yes
+
+        return reply
+
+    def run_model_missing_cobol_supplies(self):
+        self.progress.setVisible(True)
+        self.logger.info(__name__, "Running Missing Cobol Supplies model...")
+
+        try:
+            self.output_etl_missing_cobol = processing.run("model:ETL_Omisiones_Comisiones_SNC",
+                {'manzana': self.gdb_layer_paths['U_MANZANA'],
+                'predio': self.alphanumeric_file_paths['PREDIO'],
+                'rconstruccion': self.gdb_layer_paths['R_CONSTRUCCION'],
+                'rterreno': self.gdb_layer_paths['R_TERRENO'],
+                'runidad': self.gdb_layer_paths['R_UNIDAD'],
+                'uconstruccion': self.gdb_layer_paths['U_CONSTRUCCION'],
+                'uterreno':self.gdb_layer_paths['U_TERRENO'],
+                'uunidad': self.gdb_layer_paths['U_UNIDAD'],
+                'vereda': self.gdb_layer_paths['R_VEREDA'],
+                'qgis:refactorfields_1:COMISIONES_TERRENO':'TEMPORARY_OUTPUT',
+                'qgis:refactorfields_2:OMISIONES_TERRENO':'TEMPORARY_OUTPUT',
+                'qgis:refactorfields_3:COMISIONES_MEJORAS':'TEMPORARY_OUTPUT',
+                'qgis:refactorfields_4:OMISIONES_MEJORAS':'TEMPORARY_OUTPUT',
+                'qgis:refactorfields_5:COMISIONES_UNID_PH':'TEMPORARY_OUTPUT',
+                'qgis:refactorfields_6:OMISIONES_UNID_PH':'TEMPORARY_OUTPUT',
+                'qgis:refactorfields_7:COMISIONES_MZ':'TEMPORARY_OUTPUT',
+                'qgis:refactorfields_8:OMISIONES_MZ':'TEMPORARY_OUTPUT',
+                'qgis:refactorfields_9:COMISIONES_VR':'TEMPORARY_OUTPUT',
+                'qgis:refactorfields_10:OMISIONES_VR':'TEMPORARY_OUTPUT'},
+                                                           feedback=self.custom_feedback)
+        except QgsProcessingException as e:
+            if self.custom_feedback.isCanceled():
+                # The algorithm can throw errors even if canceled, so catch them and if we canceled, silent them.
+                return False, 'Missing cobol supplies algorithm canceled!'
```

Comment:
Aren't we in the SNC class?
So, make sure we don't refer to any "Cobol" stuff.

Reply:
Yes, I reviewed it and I changed in the whole class.